_posts/2024-05-14-ASUworkshop.md

```diff
@@ -0,0 +1,173 @@
+the greater Phoenix area.
+
+Note that airfare for travel awardees will have to be *booked by June 3 through
+our travel agency* so as an awardee you will have to be able to
+respond to our communications (via email) immediately and act quickly.
+
+Bursary recipients will be notified by May 29. 
+
+
+#### Application without bursary request or local attendees
+
+Register by **June 5, 2024** (midnight anywhere on Earth).
+
+
+### Travel
+
+[Tempe][] is part of the "Valley of the Sun" metropolitan area, which encompasses
+[Phoenix][], the 5th most populous city in the US. Today the area is home to more
+than 4.8 million people and encompasses the ancient lands of the Akimel O'odham
+(Pima), Maricopa, and Tohono O'odham tribes.
+
+
+#### Airport
+
+**Phoenix Sky Harbor** [PHX][] airport has direct connections to all major hubs
+in the US.
+
+PHX airport is 5.5 miles (10 minutes by car/rideshare) from the workshop venue.
+
+#### Public transport (Light Rail)
+
+The [Valley Metro Light Rail][Light Rail] connects the airport to Tempe along a
+West-East axis. If your destination is near a Light Rail stop then
+public transport works well:
+
+1. Take the free [PHX Sky Train][] from the terminal to the **44th St/Washington**
+Metro Rail stop.
+2. Walk from the Sky Train stop to the Light Rail stop (all inside the 44th St
+Sky Train structure)
+3. Buy a ticket at the ticket machine near the exit escalators to the Light
+Rail stop.
+4. At the [44th St/Washington stop][LightRail44thSt] (Stop #10018) take the
+*east-bound* Light Rail (direction Mesa, Gilbert Rd/Main St).
+5. The stop closest to the Hyatt Place Hotel and the workshop venue is
+**University Dr/Rural Rd** (Stop #10023). If you want to get off closer to
+downtown Tempe get off one stop earlier at **Veterans Way/College Ave** (Stop #10025)
+6. Walk to your destination.
+
+#### Car/rideshare
+
+To get from the airport to anywhere in Tempe either take a cab (traditional or self-driving
+Waymo) or rideshare (Lyft, Uber).
+
+
+#### Workshop Venue
+
+The workshop will be held in the Student Success Center in the Department of
+Physics at [Arizona State University][ASU] (room PSF 186 in the [Physical Sciences F
+building][PSF]). The address is
+
+  Department of Physics  
+  Arizona State University  
+  Physical Sciences F Building  
+  550 E Tyler Mall, PSF 186  
+  Tempe, AZ 85287  
+  USA
+ 
+Paid parking is available in the [Tyler Street Parking
+Structure][TylerStParking] and other on-campus parking structures.
+
+### Accommodation
+
+The closest hotel (5 min by foot) is the [Hyatt Place Tempe][HyattPlace].
+
+Other hotels are available in Tempe within walking distance.
+
+
+### Weather
+
+Arizona is *hot* during June. Temperatures in excess of 110ºF (43.3ºC) are not
+rare so it is important that you bring a hat for sun protection and carry
+enough water with you.
+
+Avoid hiking in the desert during the hot portion of the day (10am - 6pm) as it
+can be deadly.
+
+
```

Comment:
### Who to Contact

If you have any questions or special requests related to this workshop, you may [contact the organizing committee](mailto:workshops@mdanalysis.org).